docs/DeveloperGuide.md

```diff
@@ -642,6 +642,19 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Undo a command
+
+1. Undoing a `delete` command
+
+   1. Prerequisites: The previous command is a `delete` command
```

Comment:
can we add fullstop at the end of the sentence such that it is consistent with the rest?